client/my-sites/comments/index.js

```diff
@@ -7,26 +7,44 @@ import page from 'page';
 /**
  * Internal dependencies
  */
-import controller from 'my-sites/controller';
-import { clearCommentNotices, comments, redirect } from './controller';
+import { siteSelection, navigation, sites } from 'my-sites/controller';
+import { clearCommentNotices, comment, postComments, redirect, siteComments } from './controller';
 import config from 'config';
+import { VALID_STATUSES } from './constants';
 
 export default function() {
 	if ( ! config.isEnabled( 'comments/management' ) ) {
 		page( '/stats' );
 	}
 
 	if ( config.isEnabled( 'comments/management' ) ) {
-		page( '/comments/:status?', controller.siteSelection, redirect, controller.sites );
+		// Site View
+		page(
+			`/comments/:status(${ VALID_STATUSES.join( '|' ) })/:site`,
+			siteSelection,
+			navigation,
+			siteComments
+		);
 
+		// Post View
 		page(
-			'/comments/:status/:site',
-			controller.siteSelection,
-			redirect,
-			controller.navigation,
-			comments
+			`/comments/:status(${ VALID_STATUSES.join( '|' ) })/:site/:post`,
+			siteSelection,
+			navigation,
+			postComments
 		);
 
+		// Comment View
+		page( '/comment/:site/:comment', siteSelection, navigation, comment );
```

Comment:
Let's add two feature flags here for post/comment views. We can remove them as soon as content is ready.